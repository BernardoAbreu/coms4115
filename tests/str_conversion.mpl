a = 1234;
b = 3.1415;
c = [8, 9, 1, 10];
d = true;

print(str(a));
print("\n");

print(str(b));
print("\n");

print(str(c));
print("\n");

print(str(d));
print("\n");

tuple Tup {a:integer, b, c:real}

t = Tup;

t$a = 5;
t$b = "hello";
t$c = 3.131535;

print(str(t));
print("\n");

e = [1, 2, 3, 4, 5];

f = [c, e];

print(str(f));
print("\n");