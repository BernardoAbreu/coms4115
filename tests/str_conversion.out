1234
3.1415
[ 8, 9, 1, 10 ]
true
(5, hello, 3.131535)
[ [ 8, 9, 1, 10 ][ 1, 2, 3, 4, 5 ] ]
